examples/dataproc-gcs-connector/terraform/main.tf

```diff
@@ -0,0 +1,115 @@
+# Copyright 2019 Google Inc.
```

Comment:
Add a README.md in this dir explaining how to use it.

You can use [terraform-docs](https://github.com/segmentio/terraform-docs) to auto generate a table for your inputs based on variables.